.gitignore

```diff
@@ -25,8 +25,10 @@
 .eslintrc.json
 .sscs-lint.yml
 .env.local
+.env.development
```

Comment:
The `.env.development` file must be tracked. In your local development environment you should work with `.env.local` instead.